experimenter/experimenter/experiments/models.py

```diff
@@ -798,6 +798,29 @@ def allocate_bucket_range(self):
             ),
         )
 
+    @property
+    def feature_has_live_multifeature_experiments(self):
+        """Live multifeature experiments that share a feature with this experiment.
+
+        Returns:
+            A list of experiment slugs.
+        """
+        matching = set()
+        live_experiments = NimbusExperiment.objects.filter(
+            status=self.Status.LIVE,
+            application=self.application,
+        )
+        if len(live_experiments.all()) >= 0:
+            for feature in self.feature_configs.all().values_list("slug", flat=True):
+                experiments = (
+                    live_experiments.annotate(n_feature_configs=Count("feature_configs"))
+                    .filter(n_feature_configs__gt=1)
+                    .filter(feature_configs__slug=feature)
```

Comment:
Oh you can get rid of the loop if you do

```py
feature_slugs = self.feature_configs.all().values_list("slug", flat=True)
...
    .filter(feature_configs__slug__in=feature_slugs)
```

Reply:
`__slug__in` -- that's exactly what I was looking for!